.github/workflows/ci.yml

```
@@ -0,0 +1,75 @@
+name: CI
+
+on:
+  push:
+    branches: "master"
+    tags: ["*"]
+  pull_request:
+  release:
+
+jobs:
+  test:
+    name: Julia ${{ matrix.julia-version }} - ${{ matrix.os }} - ${{ matrix.julia-arch }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        julia-version:
+          - "1.0"
+          - "1"
+          - "nightly"
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        julia-arch:
+          - x64
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: ${{ matrix.julia-version }}
+          arch: ${{ matrix.julia-arch }}
+      - name: Cache artifacts
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-artifacts
+        with:
+          path: |
+            ~/.julia/artifacts
+            ~/.julia/registries
+          key: ${{ runner.os }}-test-${{ env.cache-name }}-${{ hashFiles('**/Project.toml') }}
+          restore-keys: |
+            ${{ runner.os }}-test-${{ env.cache-name }}-
+            ${{ runner.os }}-test-
+            ${{ runner.os }}-
+      - uses: julia-actions/julia-buildpkg@latest
+      - uses: julia-actions/julia-runtest@latest
+      - uses: julia-actions/julia-uploadcodecov@v0.1
+        continue-on-error: true
+      - uses: julia-actions/julia-uploadcoveralls@v0.1
+        continue-on-error: true
+
```

I've also been adding a job to CI for aqua tests to catch things like missing compat, etc. if you want to add it! Here's a copy from another [JuliaAstro repo](https://github.com/JuliaAstro/LACosmic.jl/blob/76c67ff6a67147c5655538022f0220fac3993a19/.github/workflows/CI.yml#L52-L62) 
```suggestion
  aqua:
    name: Aqua tests
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: julia-actions/setup-julia@v1
        with:
          version: '1'
      - uses: julia-actions/julia-buildpkg@v1
      - run: julia -e 'using Pkg; Pkg.add("Aqua")'
      - run: julia --project=@. -e 'using Aqua, UnitfulAstro; Aqua.test_all(UnitfulAstro, ambiguities=false)'
```